README.md

```diff
@@ -59,6 +59,7 @@ The benchmark can be configured through the following environment variables:
 |GRPC_CLIENT_CONCURRENCY|Number of requests to run concurrently. It can't be smaller than the number of connections.|1000|
 |GRPC_CLIENT_QPS|Rate limit, in queries per second (QPS).|0 (*unlimited*)|
 |GRPC_CLIENT_CPUS|Maximum number of cpus used by the client.|1|
+|GRPC_IMAGE_NAME|Name of Docker image built by `./build.sh`|`'grpc_bench'`|
```

Comment:
Not sure if I get it right - both Rust and Java image will have the same name, but a different tag? And in general it's transparent to someone running all the benchmarks locally?

Reply:
Yes and yes. And tag names are exactly the ones listed above along with sizes.